Mathlib/Tactic/ToAdditive/Frontend.lean

```diff
@@ -1239,8 +1242,19 @@ def elabToAdditive : Syntax → CoreM Config
         Instead, you can write the attributes in the usual way."
     trace[to_additive_detail] "attributes: {attrs}; reorder arguments: {reorder}"
     let doc ← doc.mapM fun
-      | `(str|$doc:str) => do
-        -- TODO: deprecate `str` docstring syntax in Mathlib
+      | `(str|$doc:str) => open Linter in do
+        -- Deprecate `str` docstring syntax
```

Comment:
This should have `(since := 2025-08-12)` in a comment, since there is nowhere else to put this information. No need to stop bors to add it, it can be a follow-up PR.

Reply:
#28315 (coincidentally, a permutation of this PR's digits!)